tests/pipeline/test_earnings.py

```diff
@@ -372,14 +375,20 @@ def test_infer_timestamp(self):
             infer_timestamps=True,
         )
         self.assertEqual(
-            loader.announcement_dates.keys(),
+            loader.events_by_sid.keys(),
             announcement_dates.keys(),
         )
         assert_series_equal(
-            loader.announcement_dates[0],
-            pd.Series(index=[dtx[0]] * 10, data=dtx),
+            pd.Series(loader.events_by_sid[0][ANNOUNCEMENT_FIELD_NAME]),
```

Comment:
I don't think we need to wrap this in `pd.Series`. If events_by_sid is a dataframe, then indexing by a column should return a series. I also recomend using `.loc` acessing here like `df.loc[:, ANNOUNCEMENT_FIELD_NAME]`. This makes it clear that we are looking up by column label instead of index label.